docs-src/cloud/tcld/namespace/create.md

```diff
@@ -0,0 +1,96 @@
+---
+id: create
+title: tcld namespace create
+sidebar_label: create
+description: How to create information about a Namespace in Temporal Cloud using tcld.
+tags:
+  - tcld
+---
+
+The `tcld namespace create` command creates a Temporal [Namespace](/concepts/what-is-a-namespace) in Temporal Cloud.
+
+Alias: `c`
+
+`tcld namespace create`
+
+The following modifiers control the behavior of the command.
+
+### `--request-id`
+
+The request-id to use for the asynchronous operation, if not set the server will assign one (optional).
+
+Alias: `-r`
+
+### `--ca-certificate`
+
+The base64 encoded ca certificate.
+
+Alias: `-c`
+
+### `--namespace`
+
+Specify the namespace hosted on Temporal Cloud. If not specified, the value of the environment variable `$TEMPORAL_CLOUD_NAMESPACE` is used.
```

Comment:
I think I changed this to be

> The namespace hosted on temporal cloud

We aren't supporting env variable for this because i thought it would be error prone (since usually it's used for updates)